src/Employer.Web/Orchestrators/INewVacancyOrchestrator.cs

```diff
@@ -0,0 +1,10 @@
+﻿using Esfa.Recruit.Employer.Web.ViewModels.NewVacancy;
```

Comment:
There probably is an argument for not having the interface for an orchestrator as they are tightly coupled to the controller, i.e. not very likely we'll want to use them anywhere else.

Although may be useful if unit testing at the controller level for any reason.